ansible/deploy-node-exporter.yml

```diff
@@ -0,0 +1,20 @@
+---
+- include: ansible-version.yml
+
+#- hosts: monitored
+#  become: false
+#  remote_user: root
+#  gather_facts: true
+#  roles:
+#    - role: adm
+#      adm_passwd:
+#        - "{{ passwd.art }}"
+#        - "{{ passwd.darkk }}"
+
+- hosts: monitored
+  gather_facts: true
+  vars:
+    node_exporter_listen_address: "127.0.0.1:9001"
+  roles:
+    - node_exporter
+    - node_exporter_nginx
```

Comment:
I agree, however I would suggest doing that only after we have tested that everything works as expected on a handful of nodes. At that point we should re-run dom0 with `targets: all`.